src/Place/ReadModel/RDF/RdfProjector.php

```diff
@@ -40,14 +51,24 @@ final class RdfProjector implements EventListener
     private const PROPERTY_IDENTIFICATOR_LOKALE_IDENTIFICATOR = 'generiek:lokaleIdentificator';
     private const PROPERTY_IDENTIFICATOR_VERSIE_ID = 'generiek:versieIdentificator';
 
+    private const PROPERTY_ADRES_STRAATNAAM = 'locn:thoroughfare';
+    private const PROPERTY_ADRES_HUISNUMMER = 'locn:locatorDesignator';
+    private const PROPERTY_ADRES_POSTCODE = 'locn:postcode';
+    private const PROPERTY_ADRES_GEMEENTENAAM = 'locn:postName';
+    private const PROPERTY_ADRES_LAND = 'locn:adminUnitL1';
+    private const PROPERTY_ADRES_VOLLEDIG_ADRES = 'locn:fullAddress';
+
     public function __construct(
         MainLanguageRepository $mainLanguageRepository,
         GraphRepository $graphRepository,
-        IriGeneratorInterface $iriGenerator
+        IriGeneratorInterface $iriGenerator,
+        AddressParser $addressParser
     ) {
         $this->mainLanguageRepository = $mainLanguageRepository;
         $this->graphRepository = $graphRepository;
         $this->iriGenerator = $iriGenerator;
+        $this->addressParser = $addressParser;
+        $this->addressFormatter = new FullAddressFormatter();
```

Comment:
You could make this a constructor argument with `FullAddressFormatter` as a default

Reply:
I deliberately chose not do that because we have agreed on a specific format for the `fullAddress` predicate, and only the `FullAddressFormatter` returns that format. `LocalityAddressFormatter` on the other hand, for example, does not return the address in the format that the RDF projection requires.

Reply:
Ok clear 🙂